servicetalk-client-api/src/main/java/io/servicetalk/client/api/DefaultAutoRetryStrategyProvider.java

```diff
@@ -70,6 +77,19 @@ public Builder disableWaitForLoadBalancer() {
             return this;
         }
 
+        /**
+         * By default, automatic retries waits for the associated {@link LoadBalancer} to be ready or
+         * {@link ServiceDiscoverer} to emit an error before triggering a retry for requests. This method allows a retry
+         * strategy to ignore errors from {@link ServiceDiscoverer} and wait for {@link LoadBalancer} forever.
+         *
+         * @return {@code this}.
+         * @see #disableWaitForLoadBalancer()
+         */
+        public Builder ignoreSdErrors() {
```

Comment:
Better not to use short-forms in method names => `ignoreServiceDiscovererErrors()`